.pre-commit-config.yaml

```diff
@@ -3,10 +3,17 @@
 # Running "make format" fixes most issues for you
 repos:
   - repo: https://github.com/ambv/black
-    rev: stable
+    rev: 20.8b1
```

Comment:
Where is this magic number coming from?

Reply:
1. Example on https://black.readthedocs.io/en/stable/version_control_integration.html
2. Referred to: _Replace by any tag/version:_ https://github.com/psf/black/tags
3. That link shows `20.8b1` to be the most recent version.
4. In addition that number matches the revision used in the `.pre-commi-config.yaml` of `schema-tools`

The previously used `stable` is much more magic. It's a mutable reference (by virtue of being a branch). `pre-commit` only inspects it once during installation. And even when the branch `stable` has progressed, `pre-commit` will never update it again. This means that you and I might have different versions of the `stable` black as a pre-commit hook. Explicit an actual version tags prevent this.

Reply:
:facepalm: I misread hook definition and thought this was related to schema-tools repo.